examples/flask/app.py

```diff
@@ -2,38 +2,64 @@
 # generate documentation from this file. You're probably better served reading
```

Comment:
This can be ignored, it's the same as it was previously, I just renamed app_v3.py to app.py.